src/scales/scale.time.js

```diff
@@ -361,27 +361,55 @@ function generate(min, max, capacity, options) {
  * Returns the right and left offsets from edges in the form of {left, right}.
```

Comment:
Can you clarify more what this offset it. What is it that gets offset and why does it need to be offset?

Reply:
Does this make sense?
```
/**
 * Returns the right and left offsets from edges in the form of {left, right}
 * where each values is a relative width to the scale and ranges between 0 and 1.
 * They add extra margins on the both sides by scaling down the original scale.
 * Offsets are typically used for bar charts. They are calculated based on intervals
 * between the data points to keep the leftmost and rightmost bars from being cut.
 * Offsets are added when the `offset` option is true.
 */
```

Reply:
Super helpful!

To clarify:
Are these offsets for each bar or only the leftmost and rightmost bar? I.e. can you update "edges" to specify the edge of the chart or the edge of the category

Reply:
Ok, it's "the edges of the chart".